src/api/app/models/package.rb

```diff
@@ -1374,4 +1361,19 @@ def file_exists?(filename)
   def has_icon?
     file_exists?("_icon")
   end
+
+  private
+
+  # is called before_update
+  def update_activity
+    # the value we add to the activity, when the object gets updated
+    addon = 10 * (Time.now.to_f - updated_at_was.to_f) / 86400
+    addon = 10 if addon > 10
+    new_activity = activity + addon
+    new_activity > 100 ? 100 : new_activity
+
+    self.activity_index = new_activity
+    # just for beauty
+    self.update_counter += 1
```

Comment:
we are checking the value of `update_counter` in our tests, I guess that it is because this value is important for something. If not we can remove this value and all the tests.